lib/appLauncher.js

```diff
@@ -5,11 +5,8 @@ const { existsSync } = require('fs');
 const { get } = require('http');
 const httpGet = get;
```

Comment:
@tizentv/tools need to be added on package.json.

Reply:
done